test/screenshot/lib/storage.js

```diff
@@ -137,6 +148,8 @@ class UploadableFile {
     destinationRelativeFilePath,
     fileContent,
     userAgent = null,
+    queueIndex = 0,
```

Comment:
Does it make sense for these to have defaults? Should we not be defining this every time? (I see we don't in `uploadScreenshotImage_` and `uploadDiffReport`)

Reply:
Done